### DISCUSSION TITLE
LG摺疊螢幕裝置

### DISCUSSION THREAD

**Discussion by BingRong1217** [1101-GS3073-C/Python-Community-GS3073-C#651] [General]
近年來有很多家公司都有生產可折疊的手機，但現實生活中還蠻少看到有人使用，不知道好不好用
[https://tw.news.yahoo.com/lg-display-%E5%B8%B6%E4%BE%86-360-%E5%BA%A6%E5%8F%AF%E6%91%BA%E7%9A%84-070001370.html](url)

**Comment by RogelioKG** [comment]
補一下連結：
[LG Display 帶來 360 度可摺的 OLED 面板，往外、往內都能摺](https://tw.news.yahoo.com/lg-display-ces-2023-foldable-laptop-phone-070001538.html)

感覺折疊手機從 2018 年到現在有明顯的進步，
從塑膠螢幕換成柔性玻璃、
圖形使用者介面的設計支持、
盡力讓螢幕摺痕減小等等。
折疊式螢幕的應用場景蠻廣的，
等到技術成熟了，應該就能再吸引更多使用者。

**Comment by larry40305** [comment]
我覺得沒什麼人在用的其中一個原因應該是很貴。

**Comment by bww0401** [comment]
我認為普及率不高的原因，主要是因為蘋果公司沒有出摺疊的智慧型手機吧，現在蘋果手機在市面上還是佔多數，感覺大眾在選擇新手機的考量順序，是品牌大於手機效能的。

**Comment by Aerofe** [comment]
折疊手機目前的價格還是偏高，而且對大部分人手機只是通訊的設備且能夠方便攜帶，屏幕也不需要這麼大，除非以後折疊手機的厚度能降低到比一般手機還低，不然我認為他的市佔率還是不會提升多大

**Comment by 0212Jerry** [comment]
之前看韓劇看過很多折疊手機，但好像僅限於三星。未來摺疊機的潮流不知道會不會延續，而我個人覺得蘋果現階段還不會出摺疊機，因為光apple car 跟AR VR 頭戴裝置就夠蘋果操心了。

**Comment by Vincent20030314** [comment]
我認為摺疊手機確實是不錯的發明，但隨之來的就是昂貴的價格，除此之外手機的故障機率也更高，但還是很期待之後的發展。

**Comment by Valentinaa0214** [comment]
個人雖說不是很喜歡摺疊機的設計，但讓手機變得精巧便攜的確是不錯的發明。
之前三星出的摺疊機常傳出螢幕出現摺痕或壞掉的災情，看來這項技術目前可能還不算非常成熟

**Comment by yjpxi** [comment]
可折疊手機的價格通常比傳統智能手機要高很多，這使得它們對於大多數消費者來說並不容易負擔。高昂的價格主要來自於研發成本、複雜的製造過程以及新型材料的使用。
